app/src/models/AccountSettings.ts

```diff
@@ -0,0 +1,10 @@
+/* schema describing the customizable properties of the theme */
+/* first defined at account onboarding */
+
+export interface AccountSettings {
+    accountId: string
+    accountName: string
```

Comment:
this field may become an reference by ID rather than URI to support dev/qa/prod environments, pending discussion of user story for modifying this attribute. We will defer that discussion to post-MVP